src/context/UserProvider.tsx

```diff
@@ -13,9 +13,9 @@ import { initializeApollo } from '@/apollo/apolloClient';
 import { GET_USER_BY_ADDRESS } from '@/apollo/gql/gqlUser';
 import { compareAddresses, showToastError, signMessage } from '@/lib/helpers';
 import { getToken } from '@/services/token';
-import useWallet from '@/hooks/walletHooks';
```

Comment:
this file is not used anymore, so I think you can remove it completely.

Reply:
exactly ;)
Thank you @RamRamez